odos-ui/src/components/Visualization/index.tsx

```diff
@@ -1,33 +1,51 @@
-// import styled from "@emotion/styled";
-// import { useEffect } from "react";
-// import { useQuery } from "react-query";
-// import { Dendogram, NewGraph } from "..";
-// import { config } from "../../config";
-// const D3Container = styled.div`
-//   height: calc(100% / 2);
-//   width: calc(100vw - 32px);
-//   .link {
-//     fill: none;
-//     stroke: #ccc;
-//     stroke-width: 1.5px;
-//   }
-//   margin-top: 32px;
-// `;
+// import { useState } from "react";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { RadialTree } from './RadialTree';
```

Comment:
There's a small casing typo. Import statement should be:
`import { RadialTree } from './radialTree';` or should rename the file to be `RadialTree.js`